frontend/src/stores/user-store.ts

```diff
@@ -56,11 +57,10 @@ export const useUserStore = defineStore('user', () => {
 
   /**
    * Retrieve the current signed url and update store
-   * @param {function} fetch preconfigured API fetch function
-   * @return {boolean}
+   * @param {FetchSignature} call preconfigured API fetch function
    */
-  const updateSignedUrl = async (fetch) => {
-    const { error, data: sigData } = await fetch('me/signature').get().json();
+  const updateSignedUrl = async (call: FetchAny): Promise<Error> => {
```

Comment:
I'm not very happy to use `FetchAny`, but since we're piping the call function to different methods, we can't have another type signature here. Or at least I currently don't see how. I kept the jsdoc comment param type with the appropriate type. Occurs several times in this store.

Reply:
This is fixed now since we have response types.